roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -1039,6 +1039,27 @@ media_store_path: "/matrix-media-store-parent/{{ matrix_synapse_media_store_dire
 #    config:
 #       directory: /mnt/some/other/directory
 
+# Matrix S3 Storage Provider
+{% if matrix_synapse_s3_media_store_enabled %}
+media_storage_providers:
+- module: s3_storage_provider.S3StorageProviderBackend
+  # In the context of this module the local store means the S3 Bucket
+  # and the remote store the local volume ¯\_(ツ)_/¯ (i swaped it for convenience)
+  store_local: {{ matrix_synapse_s3_media_store_remote }}
+  store_remote: {{ matrix_synapse_s3_media_store_local }}
+  store_synchronous: {{ matrix_synapse_s3_media_store_sync }}
```

Comment:
This does not follow our naming conventions (`matrix_SERVICE_SETTING`).

It should be called `matrix_synapse_media_storage_provider_s3_store_synchronous` instead.

The same applies to various other options as well (`region` -> `region_name`, `access` -> `access_key_id`, etc.).

Reply:
All vars renamed.